Core/include/Acts/Vertexing/HelicalTrackLinearizer.hpp

```diff
@@ -38,46 +41,18 @@ namespace Acts {
 /// This class computes A and B using the analytic formulae of Ref. (1).
 ///
 /// Ref. (1) - CERN-THESIS-2010-027, Giacinto Piacquadio (Freiburg U.)
-///
-/// @tparam propagator_t Propagator type
-/// @tparam propagator_options_t Propagator options type
-template <typename propagator_t,
-          typename propagator_options_t = PropagatorOptions<>>
 class HelicalTrackLinearizer {
  public:
-  using Propagator_t = propagator_t;
-
   /// State struct
-  struct State {
-    /// @brief The state constructor
-    ///
-    /// @param fieldCacheIn The magnetic field cache
-    State(MagneticFieldProvider::Cache fieldCacheIn)
-        : fieldCache(std::move(fieldCacheIn)) {}
-    /// Magnetic field cache
-    MagneticFieldProvider::Cache fieldCache;
-  };
+  struct State {};
```

Comment:
is this used somewhere or can we drop it completely?

Reply:
Hm, it's still part of the `LinearizerConcept`, which is about to become obsolete anyway. I can remove the State from the concept here. The `NumericalTrackLinearizer` does not fulfill that concept anyway.